packages/core/src/components/reviews/ReviewModal/ReviewModal.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  Button,
```

Comment:
```suggestion
  Button as UIButton,
```

it's just to identify that this component is also from the UI package.